webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,15 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand")
```

Comment:
`.one("focus.js-autoExpand", "textarea.js-autoExpand")` isn't doing anything. It can be removed.